packages/insomnia-components/components/sidebar/sidebar-responses.js

```diff
@@ -20,27 +20,36 @@ const handleClick = items => {
 // Implemented as a class component because of a caveat with render props
 // https://reactjs.org/docs/render-props.html#be-careful-when-using-render-props-with-reactpurecomponent
 export default class SidebarResponses extends React.Component<Props> {
-  renderBody = (filter: string) => (
-    <div>
-      {Object.keys(this.props.responses).map(response => (
-        <React.Fragment key={response}>
-          {response.toLowerCase().includes(filter.toLocaleLowerCase()) && (
-            <SidebarItem>
-              <div></div>
-              <div>
-                <SvgIcon icon={IconEnum.indentation} />
-              </div>
-              <span onClick={() => handleClick([response])}>
-                <Tooltip message={this.props.responses[response].description} position="right">
-                  {response}
-                </Tooltip>
-              </span>
-            </SidebarItem>
-          )}
-        </React.Fragment>
-      ))}
-    </div>
-  );
+  renderBody = (filter: string): null | React.Node => {
+    const filteredValues = Object.keys(this.props.responses).filter(response =>
+      response.toLowerCase().includes(filter.toLocaleLowerCase()),
+    );
+
+    if (!filteredValues.length) {
+      return null;
+    }
+
+    return (
+      <div>
+        {filteredValues.map(response => (
+          <React.Fragment key={response}>
+            {response.toLowerCase().includes(filter.toLocaleLowerCase()) && (
+              <SidebarItem>
```

Comment:
This can be simplified now

```suggestion
        {filteredValues.map(response => (
          <SidebarItem key={response}>
```